### DISCUSSION TITLE
ccf-validation-tools/Lung/change_suggestions/

### DISCUSSION THREAD

**Discussion by giscus** [hubmapconsortium/ccf-validation-tools#240] [General]
# ccf-validation-tools/Lung/change_suggestions/

https://hubmapconsortium.github.io/ccf-validation-tools/Lung/change_suggestions/

**Comment by gpryhuber** [comment]
GP Comments on AS-AS
AS-AS
1 - Secondary Pulmonary Lobule --> terminal bronchiole
secondary pulmonary lobule should have a 'has part' some 'terminal bronchiole' PMID:3259815
agree
2 - subsegmental bronchus --> cartilage of bronchus
It needs a 'has part' relationship.
agree
3 - pulmonary alveolar duct --> alveolus of lung
This relationship is difficult to validate, because single alveolus can be found as scattered outpockets of the respiratory bronchiole. I would recommend to have the alveolus of lung in the next column of pulmonary acinus. See issue 286
GP: did make some modifications to Lung ASCT+B table, please check if we fix this with these modifications?
Secondary pulmonary lobule -> terminal bronchiole —> with cells and ECM that compose that structure

Secondary pulmonary lobule ->terminal bronchiole -> pulmonary acinus -> respiratory bronchiole —> with the cells and ECM that compose the respiratory bronchiole

Secondary pulmonary lobule ->terminal bronchiole -> pulmonary acinus -> respiratory bronchiole -> alveolus of respiratory bronchiole (new term - or leave as alveolus of lung but this will come up again as part for the pulmonary acinus) —> with the cells and ECM that compose that simple alveolar structure

Secondary pulmonary lobule ->terminal bronchiole -> pulmonary acinus -> alveolar duct (that are vital to but not part for respiratory bronchioles) —> with the cells and ECM that compose the alveolar ducts

Secondary pulmonary lobule ->terminal bronchiole -> pulmonary acinus -> alveolus of lung ->pulmonary alveolar parenchyma —> with the cells and ECM that compose the parenchyma

4 - subsegmental bronchus --> lobar bronchus mesenchyme
It needs a new term (either general bronchus mesenchyme and add 'has part' axiom, or subsegmental bronchus mesenchyme). 
GP Would prefer “mesenchyme of subsegmental bronchus” or “subsegmental bronchus mesenchyme”
GP And likewise “epithelium of subsegmental bronchus” and “cartilage of subsegmental bronchus” and “smooth muscle of subsegmental bronchus”
5 - subsegmental bronchus --> bronchus smooth muscle
It needs a 'has part' relationship. – 
GP See above #4 “smooth muscle of subsegmental bronchus”
6 - lung --> immune system
No actions for now. 
GP Agreed
7 - epithelium of segmental bronchus --> bronchus submucosal gland
The bronchus submucosal gland are not part of an epithelium. I can add a 'has part' some 'bronchus sumucosal gland' to segmental bronchus, and this term replaces epithelium of segmental bronchus.
GP: agree segmental bronchus ‘has part’ bronchus submucosal gland
Also lobar bronchus ‘has part’ bronchus submucosal gland
Also subsegmental bronchus ‘has part’ bronchus submucosal gland’ 
Or be more specific as in “submucosal gland of segmental bronchus”, “submucosal gland of lobar bronchus” “submucosal gland of subsegmental bronchus” and “submucosal gland of main bronchus”
8 - pulmonary alveolar duct --> pulmonary alveolar parenchyma
pulmonary alveolar parenchyma is parenchyma and (part of some alveolus of lung). Therefore, mapping to alveolus of lung instead of pulmonary alveolar duct will validate.
GP: believe I corrected this
9 - Tracheobronchial tree --> pulmonary nerve plexus
It should have a part of.
GP: agreed
10 - lobar bronchus --> bronchus submucosal gland
It needs a 'has part' relationship.
GP: agreed, or add lobar bronchus of submucosal gland (same for #11, 12, 13,,and similar)
11 - lobar bronchus --> bronchus connective tissue
There is a lobar bronchus connective tissue UBERON:0003591.
GP: now used
12 - lobar bronchus --> bronchus smooth muscle
It needs a 'has part' relationship.
GP: agreed
13 - lobar bronchus --> cartilage of bronchus
It needs a 'has part' relationship.
14 - Segmental Bronchus --> bronchus smooth muscle
It needs a 'has part' relationship.
GP: or new term ‘smooth muscle of segmental bronchus’
25 - Segmental Bronchus --> cartilage of bronchus
I can add for both structures a 'has part' some 'cartilage of bronchus' axiom (UBERON:0001956) (also for subsegmental bronchus).
GP: or ‘cartilage of segmental bronchus’?
16 - Segmental Bronchus --> lobar bronchus mesenchyme
This can't validate, as the lobar bronchus mesenchyme should map to lobar bronchus only. This problem will also occur for subsegmental bronchus once Ubergraph gets the new release. I can add for both structures a 'has part' some 'lung mesenchyme' axiom (UBERON:0004883) and that terms substitutes lobar bronchus mesenchyme.
GP: as for #4 above, GP Would prefer “mesenchyme of segmental bronchus” or “segmental bronchus mesenchyme”
17 - subsegmental bronchus --> epithelium of segmental bronchus
The correct term would be epithelium of bronchus (UBERON:0002031). It needs a has part axiom. However, in this case a new term might be beneficial.
GP: as for #4 above, GP And likewise “epithelium of subsegmental bronchus” and “cartilage of subsegmental bronchus” and “smooth muscle of subsegmental bronchus”
18 - subsegmental bronchus --> primary bronchiole
Primary bronchiole needs to be edited. This is tricky, because according of some literature, there is a ssub-subsegmental bronchus, and therefore any axiom would be not true.
GP: Maybe modify definition to say something like: “First bronchiole that arises from a bronchus”. Proximally connected to a bronchus.  There are actually probably 5-6 generations of (sub-sub…sub) bronchus before reaching bronchioles.  I had these reflected in my original lung table but came to believe it would not pass ontological muster and so removed.
19 - respiratory bronchiole --> pulmonary alveolar duct
There should be a connected to relationship.
GP: agreed
20 - pulmonary alveolar parenchyma --> pulmonary capillary
Not sure of the expected relationship expected.
GP: would this be pulmonary alveolar parenchyma ‘has part’ pulmonary capillary and pulmonary capillary is part of pulmonary alveolar parenchyma?
21 - pulmonary lymphatic vessel --> respiratory system lymphatic vessel endothelium
It needs a 'has part' relationship.
GP: agreed
22 - immune system --> Pulmonary Acinus
We are still working for a solution for the immune system, but pulmonary acinus (and all child terms) are not part of the immune system. I would recommend to delete it or 'immune system' as last AS column.
GP: done

**Reply by aleixpuigb** [reply]
AS-AS
1 - Secondary Pulmonary Lobule --> terminal bronchiole
secondary pulmonary lobule should have a 'has part' some 'terminal bronchiole' PMID:3259815
agree
> AP: ticket in progress to add 'part of' relationship, as it is more adequate.

2 - subsegmental bronchus --> cartilage of bronchus
It needs a 'has part' relationship.
agree
> AP: Done

3 - pulmonary alveolar duct --> alveolus of lung
This relationship is difficult to validate, because single alveolus can be found as scattered outpockets of the respiratory bronchiole. I would recommend to have the alveolus of lung in the next column of pulmonary acinus. See issue 286
GP: did make some modifications to Lung ASCT+B table, please check if we fix this with these modifications?
Secondary pulmonary lobule -> terminal bronchiole —> with cells and ECM that compose that structure

> AP: agree

Secondary pulmonary lobule ->terminal bronchiole -> pulmonary acinus -> respiratory bronchiole —> with the cells and ECM that compose the respiratory bronchiole

> AP:Agree

Secondary pulmonary lobule ->terminal bronchiole -> pulmonary acinus -> respiratory bronchiole -> alveolus of respiratory bronchiole (new term - or leave as alveolus of lung but this will come up again as part for the pulmonary acinus) —> with the cells and ECM that compose that simple alveolar structure

> AP: I need to investigate if these alveoli are part_of the respiratory bronchiole or adjacent to them to create the proper relationship.

Secondary pulmonary lobule ->terminal bronchiole -> pulmonary acinus -> alveolar duct (that are vital to but not part for respiratory bronchioles) —> with the cells and ECM that compose the alveolar ducts

> AP: agree

Secondary pulmonary lobule ->terminal bronchiole -> pulmonary acinus -> alveolus of lung ->pulmonary alveolar parenchyma —> with the cells and ECM that compose the parenchyma

> AP: agree

4 - subsegmental bronchus --> lobar bronchus mesenchyme
It needs a new term (either general bronchus mesenchyme and add 'has part' axiom, or subsegmental bronchus mesenchyme).
GP Would prefer “mesenchyme of subsegmental bronchus” or “subsegmental bronchus mesenchyme”
GP And likewise “epithelium of subsegmental bronchus” and “cartilage of subsegmental bronchus” and “smooth muscle of subsegmental bronchus”

> AP: The problem with that we can overpopulate the ontology. This specific structure will need either has part relationships to cell types, or also new specific mesenchymals cell types. However, if we use' has part’ some ‘lung mesenchyme’, all cell types will automatically validate. In this case the term 'lung mesenchyme' UBERON:0004883 can be used and it will validate.

5 - subsegmental bronchus --> bronchus smooth muscle
It needs a 'has part' relationship. –
GP See above 4 “smooth muscle of subsegmental bronchus”
> AP same as 4

6 - lung --> immune system
No actions for now.
GP Agreed
7 - epithelium of segmental bronchus --> bronchus submucosal gland
The bronchus submucosal gland are not part of an epithelium. I can add a 'has part' some 'bronchus sumucosal gland' to segmental bronchus, and this term replaces epithelium of segmental bronchus.
GP: agree segmental bronchus ‘has part’ bronchus submucosal gland
Also lobar bronchus ‘has part’ bronchus submucosal gland
Also subsegmental bronchus ‘has part’ bronchus submucosal gland’
Or be more specific as in “submucosal gland of segmental bronchus”, “submucosal gland of lobar bronchus” “submucosal gland of subsegmental bronchus” and “submucosal gland of main bronchus”
> AP: Added has part relationships to avoid overpopulation

8 - pulmonary alveolar duct --> pulmonary alveolar parenchyma
pulmonary alveolar parenchyma is parenchyma and (part of some alveolus of lung). Therefore, mapping to alveolus of lung instead of pulmonary alveolar duct will validate.
GP: believe I corrected this
> AP: It is corrected

9 - Tracheobronchial tree --> pulmonary nerve plexus
It should have a part of.
GP: agreed

10 - lobar bronchus --> bronchus submucosal gland
It needs a 'has part' relationship.
GP: agreed, or add lobar bronchus of submucosal gland (same for https://github.com/hubmapconsortium/ccf-validation-tools/pull/11, 12, 13,,and similar)
> AP: Added 'has part'

11 - lobar bronchus --> bronchus connective tissue
There is a lobar bronchus connective tissue UBERON:0003591.
GP: now used
> AP: Done

12 - lobar bronchus --> bronchus smooth muscle
It needs a 'has part' relationship.
GP: agreed
> AP: Done

13 - lobar bronchus --> cartilage of bronchus
It needs a 'has part' relationship.
> AP: Done

14 - Segmental Bronchus --> bronchus smooth muscle
It needs a 'has part' relationship.
GP: or new term ‘smooth muscle of segmental bronchus’
> AP: Added 'has part'

15 - Segmental Bronchus --> cartilage of bronchus
I can add for both structures a 'has part' some 'cartilage of bronchus' axiom (UBERON:0001956) (also for subsegmental bronchus).
GP: or ‘cartilage of segmental bronchus’?
> AP: Added 'has part' to avoid overpopulating

16 - Segmental Bronchus --> lobar bronchus mesenchyme
This can't validate, as the lobar bronchus mesenchyme should map to lobar bronchus only. This problem will also occur for subsegmental bronchus once Ubergraph gets the new release. I can add for both structures a 'has part' some 'lung mesenchyme' axiom (UBERON:0004883) and that terms substitutes lobar bronchus mesenchyme.
GP: as for 4 above, GP Would prefer “mesenchyme of segmental bronchus” or “segmental bronchus mesenchyme”
> AP: Same as 4, for now a `'has part' some 'lung mesenchyme'` has been added and it can be used. 

17 - subsegmental bronchus --> epithelium of segmental bronchus
The correct term would be epithelium of bronchus (UBERON:0002031). It needs a has part axiom. However, in this case a new term might be beneficial.
GP: as for 4 above, GP And likewise “epithelium of subsegmental bronchus” and “cartilage of subsegmental bronchus” and “smooth muscle of subsegmental bronchus”
> AP: I have added the 'has part' axiom. Same with previous, the addition of new terms will be considered.

18 - subsegmental bronchus --> primary bronchiole
Primary bronchiole needs to be edited. This is tricky, because according of some literature, there is a ssub-subsegmental bronchus, and therefore any axiom would be not true.
GP: Maybe modify definition to say something like: “First bronchiole that arises from a bronchus”. Proximally connected to a bronchus. There are actually probably 5-6 generations of (sub-sub…sub) bronchus before reaching bronchioles. I had these reflected in my original lung table but came to believe it would not pass ontological muster and so removed.
> AP: The problem of proximally connected to is that it means it is proximally connected to all bronchus, even lobar bronchus or segmental bronchus.

19 - respiratory bronchiole --> pulmonary alveolar duct
There should be a connected to relationship.
GP: agreed
> AP: The problem here is that in other mammals (example mice), they don't have respiratory bronchioles (or few), and pulmonary alveolar duct is directly connected to terminal bronchiole. Can pulmonary alveolar duct parent be `pulmonary acinus` (proposed in 3)? 

20 - pulmonary alveolar parenchyma --> pulmonary capillary
Not sure of the relationship expected.
GP: would this be pulmonary alveolar parenchyma ‘has part’ pulmonary capillary and pulmonary capillary is part of pulmonary alveolar parenchyma?
> AP: I can add the first axiom `pulmonary alveolar parenchyma SubclassOf ‘has part’ some pulmonary capillary`, but not the second one, as not all pulmonary capillaries are part of the pulmonary alveolar parenchyma.

21 - pulmonary lymphatic vessel --> respiratory system lymphatic vessel endothelium
It needs a 'has part' relationship.
GP: agreed
> AP: Done

22 - immune system --> Pulmonary Acinus
We are still working for a solution for the immune system, but pulmonary acinus (and all child terms) are not part of the immune system. I would recommend to delete it or 'immune system' as last AS column.
> GP: done

**Comment by gpryhuber** [comment]
Remainder of list reviewed:
CT-CT
1 - epithelial cell of tracheobronchial tree --> airway submucosal gland duct basal cell
This can be solved if airway submucosal gland duct basal cell is in CT1, and we include 'has part' some 'airway submucosal gland duct basal cell' to terminal ciliated ducts for tracheal submucosal gland (term under review, so I will add before it is merged).
GP: agree Will need to remember this when bring back the CT1, CT2 etc
2 - epithelial cell of tracheobronchial tree --> serous cell of epithelium of bronchus
tracheobronchial serous cell should be an 'epithelial cell', but it is not. Once this is resolved, serous cell of epithelium of bronchus will be inferred as an epithelial cell of tracheobronchial tree.
GP: agree Will need to remember this when bring back the CT1, CT2 etc
3 - epithelial cell of tracheobronchial tree --> airway submucosal gland collecting duct epithelial cell
Same approach as CT-CT1.
GP: agree Will need to remember this when bring back the CT1, CT2 etc
4 - epithelial cell of tracheobronchial tree --> mucus secreting cell of bronchus submucosal gland
Same approach as CT-CT1.
GP: agree Will need to remember this when bring back the CT1, CT2 etc
5 - pulmonary interstitial fibroblast --> tracheobronchial chondrocyte
The ID used is wrong. CL:0019002 is tracheobronchial condrocyte. lung perichondrial fibroblast is CL:4033026.
GP: corrected
6 - epithelial cell of lung --> club cell
After some disccussion wish Joshua, club cell is now an epithelial cell of tracheobronchial tree (CL:0002202). The recommendation is to use it as a parent term.
GP: Agreed. Will need to remember this when bring back the CT1, CT2 etc
7 - epithelial cell of tracheobronchial tree --> lung goblet cell
Change to tracheobronchial goblet cell.
GP: done for segmental bronchus and subsegmental bronchus; left ‘goblet cell of epithelium of lobar bronchus’ for lobar bronchus
8 - epithelial cell of lung --> lung ciliated cell
Not all ciliated cells are epithelial cells. In the lung, are all ciliated cells also epithelial cells? Yes, axiom needs to be included with reference.
GP: agree
9 - lung endothelial cell --> vein endothelial cell of respiratory system
The respiratory system is too broad, best solution is to add a new term (vein endothelial cell of lung).
GP: agree though would prefer ‘pulmonary vein endothelial cell’ to have consistency with ‘pulmonary artery endothelial cell’
10 - lung endothelial cell --> endothelial cell of respiratory system lymphatic vessel
The respiratory system is more general than lung, probably creating a new term is the best solution (lymphatic endothelial cell of lung).
GP: agree. Or ‘pulmonary lymphatic endothelial cell’?  Left as is for this table but would like new term
11 - connective tissue cell --> myofibroblast cell
A myofibroblast cell is a connective tissue cell. STATUS: DONE
GP: agree
12 - myeloid dendritic cell --> plasmacytoid dendritic cell
This relationship is tricky, as it is not clear if plasmacytoid dendritic cell has myeloid, lymphoid or both precursors. Alternatively, the following terms from the hierarchy can be used as parental terms:
hematopoietic cell > leukocyte > dendritic cell > plasmacytoid dendritic cell
GP: agree, Will need to remember this when bring back the CT1, CT2 etc
13 - myeloid leukocyte --> megakaryocyte
myeloid leukocyte should be substituted for myeloid cell(CL:0000763).
GP: if megakaryocyte is not considered an immune cell (see #33 below), it probably is also not a myeloid leukocyte.  Will need to review this when bring back the CT1, CT2 etc
CT-AS
GP: Agree with #1-12 below
1 - terminal bronchiole epithelium --> club cell
It needs a 'has part' relationship.
2 - epithelium of segmental bronchus --> club cell
It needs a 'has part' relationship.
3 - epithelium of respiratory bronchiole --> club cell
It needs a 'has part' relationship.
4 - epithelium of bronchiole --> club cell
It needs a 'has part' relationship.
5 - pulmonary alveolar parenchyma --> type I pneumocyte
It needs a 'has part' relationship.
6 - pulmonary alveolar parenchyma --> type II pneumocyte
It needs a 'has part' relationship.
7 - epithelium of segmental bronchus --> brush cell of bronchus
It needs a 'has part' relationship.
8 - epithelium of segmental bronchus --> basal epithelial cell of tracheobronchial tree
It needs a 'has part' relationship.
9 - epithelium of lobar bronchus --> basal epithelial cell of tracheobronchial tree
It needs a 'has part' relationship.
10 - epithelium of lobar bronchus --> ciliated cell of the bronchus
It needs a 'has part' relationship.
11 - epithelium of segmental bronchus --> ciliated cell of the bronchus
It needs a 'has part' relationship.
12 - bronchus smooth muscle --> bronchial smooth muscle cell
It needs a 'has part' relationship.
13 - respiratory system venous endothelium --> adult endothelial progenitor cell
The textual definition of an adult endothelial progenitor cell is the following:
An adult angioblastic cell released from the bone marrow, or from the kidney in some teleost species, capable of blood circulation and participation in angiogenesis by differentiating into blood vessel endothelial cells.
Is this cell type meant to be resident or circulating? In case it is a resident type, it would need a new term. In case it is circulating, we will need a similar soultion as immune cells.
GP: this requires further investigation and discussion with lung experts as to if (a)EPC should be considered “resident” lung cells
14 - respiratory system lymphatic vessel endothelium --> endothelial cell of respiratory system lymphatic vessel
It is missing a part of relationship.
GP:, also note suggested change of term for endothelial cell of respiratory system lymphatic vessel
(above)
15- pulmonary alveolar parenchyma --> lung pericyte
It needs a 'has part' relationship.
GP: Agree with #15-21
16 - pulmonary nerve plexus --> sympathetic neuron
It needs a 'has part' relationship.
17 - epithelium of segmental bronchus --> pulmonary ionocyte
It needs a 'has part' relationship.
18 - epithelium of lobar bronchus --> pulmonary ionocyte
It needs a 'has part' relationship.
19 - cartilage of bronchus --> tracheobronchial chondrocyte
It needs a 'has part' relationship.
20 - lobar bronchus mesenchyme --> tracheobronchial smooth muscle cell
It needs a 'has part' relationship.
21 - epithelium of segmental bronchus --> lung goblet cell
It needs a 'has part' relationship.
22 - epithelium of segmental bronchus --> lung neuroendocrine cell
It needs a 'has part' relationship. This is a very general term, are there lung neuroendocrine cell subtypes needed?
GP: at this time we have not subtyped these cells; Please note though that we need a synonym within the ‘lung neuroendocrine cell’ entry, the lung community refers to these as pulmonary neuroendocrine cells (PNECs).  We also refer to clusters of PNECs as neuroendocrine cell body (NEBs) UBERON:0010002, that definition refers to PNECs
23 - epithelium of bronchiole --> lung ciliated cell
It needs a 'has part' relationship.
GP: agree
24 - terminal bronchiole epithelium --> lung ciliated cell
It needs a 'has part' relationship.
GP: agree
25 - pulmonary capillary --> alveolar capillary type 1 endothelial cell
According to this reference it is probably missing a part of relationship.
GP: Correct
26 - pulmonary capillary --> alveolar capillary type 2 endothelial cell
According to this reference it is probably missing a part of relationship.
GP: Correct
27 - pulmonary alveolar parenchyma --> alveolar type 1 fibroblast cell
It is missing a part of relationship.
GP: agree; add synonym AF1
28 - pulmonary alveolar parenchyma --> alveolar type 2 fibroblast cell
It is missing a part of relationship. Include synonym 'resident interstitial lung fibroblasts'.
GP: This name should be modified – the fibroblast task force has determined these should be ‘adventitial fibroblast cell’, probably include synonym ‘alveolar type 2 fibroblast cell’
29 - respiratory system venous endothelium --> vein endothelial cell of respiratory system
It is missing a part of relationship.
GP: Agree though note #9 above - The respiratory system is too broad, best solution is to add a new term (vein endothelial cell of lung). GP: agree though would prefer ‘pulmonary vein endothelial cell’ to have consistency with ‘pulmonary artery endothelial cell’

30 - smooth muscle tissue of terminal bronchiole --> bronchiolar smooth muscle cell
It needs a 'has part' relationship.
GP: Agree
31 - smooth muscle tissue of bronchiole --> bronchiolar smooth muscle cell
It is missing a part of relationship.
GP: Agree
32 - smooth muscle tissue of respiratory bronchiole --> bronchiolar smooth muscle cell
It needs a 'has part' relationship.
GP: Agree
33 - immune system --> lung megakaryocyte
Megakaryocytes are not considered to be part of the immune system. It can directly map to l
GP: not sure what was intended here but I placed it in the table as has part connective tissue for now

**Reply by aleixpuigb** [reply]
CT-CT

> AP: Right now there is only one level of CT.

CT-AS
GP: Agree with 1-12 below
1 - terminal bronchiole epithelium --> club cell
It needs a 'has part' relationship.
2 - epithelium of segmental bronchus --> club cell
It needs a 'has part' relationship.
3 - epithelium of respiratory bronchiole --> club cell
It needs a 'has part' relationship.
4 - epithelium of bronchiole --> club cell
It needs a 'has part' relationship.
5 - pulmonary alveolar parenchyma --> type I pneumocyte
It needs a 'has part' relationship.
6 - pulmonary alveolar parenchyma --> type II pneumocyte
It needs a 'has part' relationship.
7 - epithelium of segmental bronchus --> brush cell of bronchus
It needs a 'has part' relationship.
8 - epithelium of segmental bronchus --> basal epithelial cell of tracheobronchial tree
It needs a 'has part' relationship.
9 - epithelium of lobar bronchus --> basal epithelial cell of tracheobronchial tree
It needs a 'has part' relationship.
10 - epithelium of lobar bronchus --> ciliated cell of the bronchus
It needs a 'has part' relationship.
11 - epithelium of segmental bronchus --> ciliated cell of the bronchus
It needs a 'has part' relationship.
12 - bronchus smooth muscle --> bronchial smooth muscle cell
It needs a 'has part' relationship.
> AP: 1-12 have been added and will appear in next release.

13 - respiratory system venous endothelium --> adult endothelial progenitor cell
The textual definition of an adult endothelial progenitor cell is the following:
An adult angioblastic cell released from the bone marrow, or from the kidney in some teleost species, capable of blood circulation and participation in angiogenesis by differentiating into blood vessel endothelial cells.
Is this cell type meant to be resident or circulating? In case it is a resident type, it would need a new term. In case it is circulating, we will need a similar soultion as immune cells.
GP: this requires further investigation and discussion with lung experts as to if (a)EPC should be considered “resident” lung cells
> AP: pending

14 - respiratory system lymphatic vessel endothelium --> endothelial cell of respiratory system lymphatic vessel
It is missing a part of relationship.
GP:, also note suggested change of term for endothelial cell of respiratory system lymphatic vessel
(above)
> AP: noted

15- pulmonary alveolar parenchyma --> lung pericyte
It needs a 'has part' relationship.
GP: Agree with 15-21
16 - pulmonary nerve plexus --> sympathetic neuron
It needs a 'has part' relationship.
17 - epithelium of segmental bronchus --> pulmonary ionocyte
It needs a 'has part' relationship.
18 - epithelium of lobar bronchus --> pulmonary ionocyte
It needs a 'has part' relationship.
19 - cartilage of bronchus --> tracheobronchial chondrocyte
It needs a 'has part' relationship.
20 - lobar bronchus mesenchyme --> tracheobronchial smooth muscle cell
It needs a 'has part' relationship.
21 - epithelium of segmental bronchus --> lung goblet cell
It needs a 'has part' relationship.
> AP: 15-21 have been added and will appear in next release.

22 - epithelium of segmental bronchus --> lung neuroendocrine cell
It needs a 'has part' relationship. This is a very general term, are there lung neuroendocrine cell subtypes needed?
GP: at this time we have not subtyped these cells; Please note though that we need a synonym within the ‘lung neuroendocrine cell’ entry, the lung community refers to these as pulmonary neuroendocrine cells (PNECs). We also refer to clusters of PNECs as neuroendocrine cell body (NEBs) UBERON:0010002, that definition refers to PNECs
>AP: Added has part relationship. Need to add Synonym.

23 - epithelium of bronchiole --> lung ciliated cell
It needs a 'has part' relationship.
GP: agree
>AP: Done, it will appear in the next release.

24 - terminal bronchiole epithelium --> lung ciliated cell
It needs a 'has part' relationship.
GP: agree
>AP: Done, it will appear in the next release.

25 - pulmonary capillary --> alveolar capillary type 1 endothelial cell
According to this reference it is probably missing a part of relationship.
GP: Correct
26 - pulmonary capillary --> alveolar capillary type 2 endothelial cell
According to this reference it is probably missing a part of relationship.
GP: Correct
27 - pulmonary alveolar parenchyma --> alveolar type 1 fibroblast cell
It is missing a part of relationship.
GP: agree; add synonym AF1
28 - pulmonary alveolar parenchyma --> alveolar type 2 fibroblast cell
It is missing a part of relationship. Include synonym 'resident interstitial lung fibroblasts'.
GP: This name should be modified – the fibroblast task force has determined these should be ‘adventitial fibroblast cell’, probably include synonym ‘alveolar type 2 fibroblast cell’
29 - respiratory system venous endothelium --> vein endothelial cell of respiratory system
It is missing a part of relationship.
GP: Agree though note https://github.com/hubmapconsortium/ccf-validation-tools/issues/9 above - The respiratory system is too broad, best solution is to add a new term (vein endothelial cell of lung). GP: agree though would prefer ‘pulmonary vein endothelial cell’ to have consistency with ‘pulmonary artery endothelial cell’
>AP: 25-29 No progress yet.

30 - smooth muscle tissue of terminal bronchiole --> bronchiolar smooth muscle cell
It needs a 'has part' relationship.
GP: Agree
31 - smooth muscle tissue of bronchiole --> bronchiolar smooth muscle cell
It is missing a part of relationship.
GP: Agree
32 - smooth muscle tissue of respiratory bronchiole --> bronchiolar smooth muscle cell
It needs a 'has part' relationship.
GP: Agree
> AP: 30-32 have been added and will appear in next release.

33 - immune system --> lung megakaryocyte
Megakaryocytes are not considered to be part of the immune system. It can directly map to l
GP: not sure what was intended here but I placed it in the table as has part connective tissue for now
> AP: This will validate in the next CL release in two weeks.
